How-to guide migration (#165)
vegaprotocol/documentation: docs/howto/submit-order.md

* **PEGGED ORDER**  
A pegged order can be specified for persistent (`TIME_IN_FORCE_GTC`, `TIME_IN_FORCE_GTT`) and non-persistent time in force (`TIME_IN_FORCE_IOC`, `TIME_IN_FORCE_FOK`). If details for a pegged order are included in the order submission message, the system will attempt to activate this feature. A pegged order is a limit order which is set as a defined distance (`offset`) from a `reference` price (such as the best bid, mid, or best offer/ask). This is the price against which the final order price is calculated. The reference price is based on the live market. There are three options for the `reference` field in a pegged order; `PEGGED_REFERENCE_MID`, `PEGGED_REFERENCE_BEST_BID` and `PEGGED_REFERENCE_BEST_ASK`. Sample scripts for submitting and amending a pegged order are available on [GitHub](https://github.com/vegaprotocol/sample-api-scripts/tree/master/submit-amend-pegged-order). 

## How do I submit an order to a Vega node?

The flow to submit a signed `SubmitOrder` message to a Vega node is as
follows:

> 1. Wallet API: Create a new wallet, or log in to an existing one.

Reviewer: ```suggestion
1. Wallet API: Create a new wallet, or connect to an existing one.
```

2. Wallet API: Create a key pair, if one has not already been created.
3. Node API: Get a list of markets, to get the market ID of a market.
4. Wallet API: Call `SignTx`. Send the `OrderSubmission` data and a public

>    key associated with the logged-in wallet. Receive a `SignedBundle` object.

Reviewer: ```suggestion
   key associated with the connected wallet. Receive a `SignedBundle` object.
```

6. Node API: Call `SubmitTransaction`. Send the `SignedBundle` object (only required if propagate set to false on `SignTx`).

Hints:
* Using the open source Go-Wallet code, clients can either run a local
  wallet API server, or incorporate the wallet code server into their own (Golang)
  application, and avoid a network round trip to a Vega Wallet API server.
* When calling `SignTx` on the Wallet API server, it is possible to set `propagate`
  to `true`. This instructs the Wallet API server to submit the signed transaction
  directly to the Vega node specified in its config file.

> ### 1. Log in to wallet and get public key

Reviewer: ```suggestion
### 1. Connect to wallet and get public key
```

> See the section on the [Wallet service](wallet.md) to learn how to log in, list keys and select a public key.  

Reviewer: ```suggestion
See the section on the [Wallet service](wallet.md) to learn how to connect to an existing wallet, list keys and select a public key.  
```
